views/benefits/oas-fr.njk

```diff
@@ -0,0 +1,26 @@
+<div class="m-5 p-5 shadow-result">
+
+  <h2 class="text-2xl font-bold mb-6" id="oas">
+    {{__("oas.header")}}
+  </h2>
+
+  <ul class="list-none list-outside pl-0">
+    {{ blueLI("paiement unique de 300 $, non imposable, si vous recevez la pension de la Sécurité de vieillesse (SV) <br><br> OU paiment unique de 500 $, non imposable, si vous recevez le supplément de revenu garanti (SRG)") }}
```

Comment:
Should the sentences in this be capitalized. I know in Figma they are lower case but is that a mistake?

Reply:
@katewilhelm Should the benefit have lower case sentences in it? 
![image](https://user-images.githubusercontent.com/87738/81884556-3533a000-9566-11ea-9a1a-11a22c0aaa04.png)

Reply:
fixed